content/gateway_guide.md

```diff
@@ -506,46 +524,37 @@ You may also enable `RequireAuth` for your cold wallet in order to use the [Auth
 
 If ACME decides to use Authorized Accounts, ACME creates an interface for users to get their Ripple trust lines authorized by ACME's cold account. After Alice has extended a trust line to ACME from her Ripple account, she goes through the interface on ACME's website to require ACME authorize her trust line. ACME confirms that it has validated Alice's identity information, and then sends a TrustSet transaction to authorize Alice's trust line.
```

Comment:
There's probably a good way to reorganize this, but it's a little tricky because we are using RequireAuth for something other than its primary purpose. (We recommend that you enable RequireAuth on hot wallets even if you don't want to use the Authorized Accounts feature, because that way your hot wallets can never accidentally issue currency unless they also accidentally authorize trust lines first.)